app/jobs/cache_buster/bust_path_job.rb

```diff
@@ -0,0 +1,9 @@
+module CacheBuster
+  class BustPathJob < ApplicationJob
+    queue_as :bust_path
+
+    def perform(path, cache_buster = CacheBuster)
+      cache_buster.bust(path)
```

Comment:
Often with params are being passed to background jobs like this they are being done as strings. I would change the default to "CacheBuster" and then use constantize with it. `cache_buster.constantize.bust(path)`

Reply:
In this case, passing a param is done to make testing easier. If we perform this change we should modify the tests too.
I am not super comfortable with `constantize` without checking what are we constantizing. Maybe, in this case, we can avoid passing an argument and use `allow(CacheBuster).to receive ...` instead of `allow(cache_buster).to receive ...` and passing it to the job.